test/docker/fs/etc/agbot/agbot2.config.tmpl

```diff
@@ -37,6 +37,10 @@
         "CheckUpdatedPolicyS": 15,
         "CSSURL": "${CSS_URL}",
         "CSSSSLCert": "/certs/css.crt",
+        "Vault": {
```

Comment:
You need to update the ./test/Makefile target agbot-docker-prereqs to set these variables in the config file that e2edev uses at runtime. I'm refering to the for loops in that target which do a textual replacement of css and exchange URLs. We need to do the same for the vault variables you added.